docs/Encyclopedia/pages/MAGE-TAB.md

```diff
@@ -11,13 +11,13 @@ MAGE-TAB archives provide information about the data collection and data process
 
 Contents of a MAGE-TAB file:
 
-* __Investigative Description Format (IDF):__ Provides general information about the study. This includes a brief description, the investigator's contact details, bibliographic references, and a text description of the protocols used in the study<sup>2</sup>.
```

Comment:
I see a lot of these are removing the bold from the beginning of each listed item.  Was that requested in one of the tickets?  If it looks better feel free to keep it.

Reply:
Dara had commented on the lack of standardization in how we format lists here: https://jira.opensciencedatacloud.org/browse/DOC-285

I prefer the list items to be bolded, so I went back and fixed them so they should be consistent now